apps/user/models/team.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import "github.com/jinzhu/gorm"
+
+type Team struct {
+	gorm.Model
+	Name        string `gorm:"type:varchar(30);not null"`
+	Description string `gorm:"type:varchar"`
```

Comment:
Why is Description field varchar here as compared to "text" field at other places ?

Reply:
varchar() and text are equivalent in postgres
To avoid confusions,
Changed all string with fixed max length to varchar(n)
All string without limits to text